src/screens/HyperSwitch/Connectors/ConnectorTableUtils.res

```diff
@@ -236,7 +236,10 @@ let connectorEntity = (path: string) => {
     ~getCell,
     ~dataKey="",
     ~getShowLink={
-      connec => `/${path}/${connec.merchant_connector_id}?name=${connec.connector_name}`
+      connec =>
+        permission === Access
+          ? `/${path}/${connec.merchant_connector_id}?name=${connec.connector_name}`
+          : ``
```

Comment:
we should somehow comeup with proper generic solution rather than writing inline  condition